iothub/device/src/InternalClient.cs

```diff
@@ -265,11 +274,12 @@ public async Task CompleteMessageAsync(string lockToken, CancellationToken cance
         }
 
         /// <summary>
-        /// Deletes a received message from the device queue
+        /// Deletes a received message from the client's service-side queue.
```

Comment:
service-side queue?

Reply:
The service is the party that maintains the queue that this operation removes an entry from

Reply:
Maybe "Deletes a received message from the service's cloud to device message queue for this client"?